introduction/introduction.md

```diff
@@ -0,0 +1,13 @@
+This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# Introduction
+
+This Learning Path gives an introduction to inner source.
+Inner source is the strategy of applying open source practices and principles to internal software development.
+This strategy results in producing the highest quality software that is responsive and nimble to the changing needs of its many internal consumers.
```

Comment:
While I do agree that InnerSource has an impact on quality, that wouldn't be the first benefit that comes to mind. How about we mention the following instead:

- reduce barriers for collaboration / make collaboration across silos efficient
- efficiently development of shared assets
- efficiently share knowledge
- increase innovation

Reply:
Kept it to one sentence, but tried to include (or hint at) these ideas:

> reduce barriers for collaboration

"wide"

> efficiently development of shared assets

"effective"

> increase innovation
 
"responsive and nimble"

Reply:
Suggest rewording first sentence so reader understands that inner source supports open source practices at the enterprise or organization level. In my opinion, the term "internal software development" is unclear. 
REVISION: 
Inner source is the strategy of applying open source practices and principles to internal software development, i.e. within an enterprise or organization. The enterprise or organization still develops proprietary software, but internally opens up its development to allow for accessibility and reuse.

Reply:
I like the thrust of this revision.  I reworded both of your sentences to incorporate the idea of what I understood you saying but with the following constraints in sentence structure.

- **1st sentence**: No extra clauses.
- **2nd sentence**: Have the active noun in the sentence be the software instead of the company.

Reply:
@gruetter @KathrynKLewis see what you think of the (proposed) incorporation of your ideas.